src/shared/features/GrandparentViews/Inbox/Inbox.tsx

```diff
@@ -77,7 +77,7 @@ const Inbox: React.FC<IInbox> = ({ posts }) => {
 
             <GridList className={classes.gridList} cols={3}>
               {posts.map((post, index: number) => (
-                <GridListTile key={post.from} onClick={() => pressEnvelope(post)} rows={1.25}>
+                <GridListTile key={post.pid} onClick={() => pressEnvelope(post)} rows={1.25}>
```

Comment:
Oh, I made the same change in the grandparent-tech-refactor branch I'm working in ha. I changed it to the map index but it really doesn't matter as long as it's unique.

Reply:
Re: indexes as keys - 

I only recently learned this myself but using "index" for a key is an anti-pattern in React(!). I know, I was surprised too, because I use indexes as keys everywhere else... 

Check out #2 in this article: https://codeburst.io/how-to-not-react-common-anti-patterns-and-gotchas-in-react-40141fe0dcd 


One big reason not to do it is because if the items get shuffled around, the index that formerly pointed at one object now points to a different object, but this article goes into greater detail about it. 

The pid is truly unique to the object it represents, so I went with pid for that reason.

Reply:
Oh cool, thanks!